daemons/gptp/common/ieee1588port.cpp

```diff
@@ -813,17 +831,14 @@ void IEEE1588Port::processEvent(Event e)
 
 	case LINKDOWN:
 		stopPDelay();
-		if (clock->getAutomotiveProfile()) {
+		if (clock->automotiveProfileEnabled() && clock->forceAsCapable()) {
 			GPTP_LOG_EXCEPTION("LINK DOWN");
-			if (!clock->getForceAsCapable()) {
-				setAsCapable(false);
-			}
 		}
 		else {
 			setAsCapable(false);
 			GPTP_LOG_STATUS("LINK DOWN");
 		}
-		if (clock->getAutomotiveTestMode()) {
+		if (clock->automotiveProfileEnabled() && clock->automotiveTestModeEnabled()) {
```

Comment:
If we decide to keep test mode logging that wasn't automotive specific previously, we should probably remove the automotiveProfileEnabled() check here.